spark/src/main/scala/org/apache/spark/sql/comet/operators.scala

```diff
@@ -586,7 +607,8 @@ case class CometHashAggregateExec(
     mode: Option[AggregateMode],
     child: SparkPlan,
     override val serializedPlanOpt: SerializedPlan)
-    extends CometUnaryExec {
+    extends CometUnaryExec
+    with PartitioningPreservingUnaryExecNode {
```

Comment:
PartitioningPreservingUnaryExecNode implements outputPartitioning for `HashAggregateExec`.